### PR TITLE
profile details

### PR DESCRIPTION
**Pull request by javorskyp**

### CODE REVIEW

File: src/pages/Profile/AddMoto.js [hunk 1]
```diff
@@ -53,13 +58,25 @@ const AddMoto = props => {
...
-      setTimeout(() => {
+      try {
+         await axios.post('/motorcycles.json', {
+          name: form.name.value,
+          description: form.description.value,
+          city: form.city.value,
+          capacity: form.capacity.value,
+          features: form.features.value,
+          status: form.status.value,
+          user_id: auth.user_id
```
Comment (Aciderr) [inline src/pages/Profile/AddMoto.js]: Please, use camel-cased "userId" - it is more natural for js

File: src/pages/Profile/AddMoto.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ const AddMoto = props => {
           capacity: form.capacity.value,
           features: form.features.value,
           status: form.status.value,
-          user_id: auth.user_id
+          user_Id: auth.user_Id
```
Comment (Aciderr) [inline src/pages/Profile/AddMoto.js]: Sorry, should be 'userId' not 'user_Id'

File: src/pages/Profile/EditMoto.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  const submit = async form => {
+    await axios.patch(`/motorcycles/${id}.json?auth=${auth.token}`, form);
+    history.push('/profil/motorcycles');
+  }
+
+  const fetchMoto = async () => {
+    const res = await axios.get(`/motorcycles/${id}.json`);
+    const motorcycleData = res.data;
+
+    delete(motorcycleData.user_id);
```
Comment (Aciderr) [inline src/pages/Profile/EditMoto.js]: Should be "userId". When you receive user_id from firebase you can transform it to userId and then in the rest places of the app please use "userId"

File: src/pages/Profile/EditMoto.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import useAuth from '../../../../hooks/useAuth';
+
+const EditMoto = props => {
+  const [auth] = useAuth();
+  const { id } = useParams();
+  const history = useHistory();
+  const [motorcycle, setMotorcycle] = useState(null);
+
+  const submit = async form => {
+    await axios.patch(`/motorcycles/${id}.json?auth=${auth.token}`, form);
```
Comment (Aciderr) [inline src/pages/Profile/EditMoto.js:15]: The logic with "auth=${auth.token}" should be placed in a more generic place. When you log in you should automatically add auth param to all requests.

Comment (javorskyp) [reply src/pages/Profile/EditMoto.js:15]: Ok, I will try to fix it

**Pull request merged**